website/pages/docs/agent/telemetry.mdx

```diff
@@ -147,9 +147,9 @@ This is a full list of metrics emitted by Consul.
 | `consul.client.api.success.catalog_register.`         | This increments whenever a Consul agent successfully responds to a catalog register request.                                                                                                                                                                                                                                                                                                                        | requests             | counter |
 | `consul.client.rpc.error.catalog_register.`           | This increments whenever a Consul agent receives an RPC error for a catalog register request.                                                                                                                                                                                                                                                                                                                       | errors               | counter |
-| `consul.client.api.catalog_deregister.`               | This increments whenever a Consul agent receives a catalog deregister request.                                                                                                                                                                                                                                                                                                                                     | requests             | counter |
-| `consul.client.api.success.catalog_deregister.`       | This increments whenever a Consul agent successfully responds to a catalog deregister request.                                                                                                                                                                                                                                                                                                                     | requests             | counter |
-| `consul.client.rpc.error.catalog_deregister.`         | This increments whenever a Consul agent receives an RPC error for a catalog deregister request.                                                                                                                                                                                                                                                                                                                    | errors               | counter |
+| `consul.client.api.catalog_deregister.`               | This increments whenever a Consul agent receives a catalog deregister request.                                                                                                                                                                                                                                                                                                                                      | requests             | counter |
```

Comment:
Hey I just noticed that this table is getting slightly cut off for me on the right end. 

Looks like it predates this PR though:
https://www.consul.io/docs/agent/telemetry#metrics-reference